OptimizelySDK/Optimizely.cs

```diff
@@ -406,7 +406,7 @@ public virtual bool IsFeatureEnabled(string featureKey, string userId, UserAttri
         /// <param name="userAttributes">The user's attributes</param>
         /// <param name="variableType">Variable type</param>
         /// <returns>string | null Feature variable value</returns>
-        public virtual string GetFeatureVariableValueForType(string featureKey, string variableKey, string userId, 
+        public virtual T GetFeatureVariableValueForType<T>(string featureKey, string variableKey, string userId, 
```

Comment:
This is an interesting improvement :)

Reply:
yeah, we actually needed to notify for every featurevariable* api, and type was required, thats why used generics here.